packages/shared-core/src/helpers/cron.ts

```diff
@@ -0,0 +1,47 @@
+import cronValidate from "cron-validate"
+
+const INPUT_CRON_START = "(Input cron: "
+const ERROR_SWAPS = {
+  "smaller than lower limit": "less than",
+  "bigger than upper limit": "greater than",
+  daysOfMonth: "'days of the month'",
+  daysOfWeek: "'days of the week'",
+  years: "'years'",
+  months: "'months'",
+  hours: "'hours'",
+  minutes: "'minutes'",
+  seconds: "'seconds'",
+}
+
+function improveErrors(errors: string[]): string[] {
+  const finalErrors: string[] = []
+
+  for (let error of errors) {
+    if (error.includes(INPUT_CRON_START)) {
+      error = error.split(INPUT_CRON_START)[0].trim()
+    }
+    for (let [key, value] of Object.entries(ERROR_SWAPS)) {
```

Comment:
NAB: could rename `key, value` to `old, new` to make the code a little easier to follow.